packages/core-events/src/module/configureEventsModule.ts

```diff
@@ -81,5 +83,57 @@ export const configureEventsModule = async ({
       const count = await Events.countDocuments(buildFindSelector(query));
       return count;
     },
+    getReport: async ({ from, to, types } = { from: null, to: null, types: null }) => {
+      const pipeline = [];
+      const matchConditions = [];
+      if (from || to) {
```

Comment:
cool you're using pipelines! it's a little bit messy though, maybe some comments would help.